javascript/chronometer.js

```diff
@@ -1,26 +1,46 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = 0
+   
   }
   startClick(callback) {
-    // ... your code goes here
+      this.intervalId = setInterval(() => this.currentTime++, 1000);
+      if(callback) {
+        callback()
+      }
   }
+
+
   getMinutes() {
-    // ... your code goes here
+    const minutes = Math.floor(this.currentTime / 60)
+    return minutes
   }
+
+
   getSeconds() {
-    // ... your code goes here
+    const seconds = this.currentTime - this.getMinutes() * 60
+    return seconds
   }
+
   twoDigitsNumber() {
```

Comment:
A shorter way to achieve the same result using a parameter and the ternary operator: 
` twoDigitsNumber (time) {`
   ` return time < 10 ? ${0}${time}` : `${time}`
`  }`